roles/custom/matrix-bridge-mautrix-bluesky/defaults/main.yml

```diff
@@ -187,6 +189,7 @@ matrix_mautrix_bluesky_registration_yaml: |
   rate_limited: false
   de.sorunome.msc2409.push_ephemeral: true
   receive_ephemeral: true
+  io.element.msc4190: {{ matrix_mautrix_bluesky_msc4190_enabled }}
```

Comment:
This and others like it are missing `| to_json`.

Reply:
done in de9128d337522f19ffc4f5877cd63fb0f7a203c9